boards/arm/nucleo_f103rb/nucleo_f103rb.dts

```diff
@@ -21,7 +21,7 @@
 	leds {
 		compatible = "gpio-leds";
 		green_led_2: led_2 {
-			gpios = <&gpioa 5 GPIO_INT_ACTIVE_HIGH>;
+			gpios = <&gpioa 5 0>;
```

Comment:
GPIO_ACTIVE_HIGH?

Reply:
Active high is the default; previous discussion reached (undocumented?) consensus that only non-default flags would be specified in devicetree.

We should really confirm and document that.  @galak @mnkp is that in fact the standard position?

Reply:
I'd personally prefer that we show the flags in dts as its more clear if you are reading it to know.

Reply:
Also in linux DT they show the flags in dts (e.g. [eds-gpio.txt](https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/leds-gpio.txt))

Reply:
OK; there are a lot in linux that are `0` but there are also a lot that are `1` so `0` may not be indicating "default".

I too think explicit is better, but could live with zero for default.  Either way we should try to be consistent.  Hopefully my recollection that somebody had strong preference for leaving it zero for default was a false memory.

Reply:
Will set `GPIO_ACTIVE_HIGH`. I prefer to be explicit as well.